grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/GrpcStatus.java

```diff
@@ -174,8 +174,8 @@ private static Throwable unwrap(Throwable t) {
     }
 
     /**
-     * Maps GRPC status codes to http status, as defined in upstream grpc-gateway
-     * <a href="https://github.com/grpc-ecosystem/grpc-gateway/blob/master/third_party/googleapis/google/rpc/code.proto">code.proto</a>.
+     * Maps GRPC status codes to http status, as defined in upstream Google APIs
+     * <a href="https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto">code.proto</a>.
```

Comment:
Could we use the URL with commit ID instead of branch name, so it's not broken anymore?